main.c

```diff
@@ -383,6 +395,38 @@ void heartbeatTask(void *pvParameters)
     }
 }
 
+#if INCLUDE_uxTaskGetStackHighWaterMark
+void stackSampleTask(void *pvParameters)
+{
+    const struct taskdef {
+        const char *name;
+        TaskHandle_t hdl;
+    } taskdefs[] = {
+        { "Ethernet", ethernetHdl },
+        { "Heartbeat", heartbeatHdl },
+        { "uart", uartHdl },
+        { "EXI", EXIHdl },
+        { NULL, NULL },
+    };
+    UBaseType_t stackWatermark;
+
+    while (true) {
+        // Sample the stack higher watermarks of all tasks
+        vTaskDelay(10000 / portTICK_PERIOD_MS);
+
+        for (const struct taskdef *td = &taskdefs[0]; td->name != NULL; td++) {
+            stackWatermark = uxTaskGetStackHighWaterMark(td->hdl);
+
+            if (stackWatermark == 0) {
+                task_print("STACK: %s: OVERFLOW\r\n", td->name);
+            } else {
+                task_print("STACK %s: %u words (%u bytes)\r\n", td->name, stackWatermark, stackWatermark / sizeof(StackType_t));

```

Comment:
Would you kindly change that to `* sizeof` instead of `/ sizeof?` Thank you very much.